pkg/util/command_util.go

```diff
@@ -354,9 +354,17 @@ Loop:
 	return nil
 }
 
-func GetUserGroup(chownStr string, env []string) (int64, int64, error) {
+func GetUserGroup(chownStr string, env []string, from string) (int64, int64, error) {
```

Comment:
```suggestion
func GetUserGroup(chownStr string, env []string, fromContext bool) (int64, int64, error) {
```

I think `string` is a bit too magical with a vague arg name like `from`. How about renaming to e.g. `fromContext bool`, or some other bool?

---

Alternative idea, what if we actually modified `chownStr` to `0:0` when instantiating the command? That would pipe it through with minimal changes.

Reply:
I like the second option! Done.